Library/Homebrew/tab.rb

```diff
@@ -60,7 +60,6 @@ def self.from_file(path)
   def self.from_file_content(content, path)
     attributes = Utils::JSON.load(content)
     attributes["tabfile"] = path
-    attributes["runtime_dependencies"] ||= []
```

Comment:
We need to be able to differentiate between "there aren't any runtime dependencies" and "runtime dependencies weren't specified" to know whether to use the fallback.